src/Pact/Types/Pretty.hs

```diff
@@ -129,6 +129,8 @@ instance (Pretty a, Pretty b) => Pretty (a, b) where
   pretty (a, b) = tupled [pretty a, pretty b]
 instance (Pretty a, Pretty b, Pretty c) => Pretty (a, b, c) where
   pretty (a, b, c) = tupled [pretty a, pretty b, pretty c]
+instance (Pretty a, Pretty b) => Pretty (Either a b) where
```

Comment:
Not sure about this, as semantic meaning is lost. Maybe a more specialized instance for the case at hand (which is ... ?)

Reply:
Removed